packages/dai-plugin-mcd/src/CdpTypeService.js

```diff
@@ -32,4 +36,86 @@ export default class CdpTypeService extends PublicService {
     assert(types.length <= 1, `${label} matches more than one cdp type`);
     assert(types.length > 0, `${label} matches no cdp type`);
   }
+
+  useMulticall(watcher) {
+    this.cdpTypes.forEach(type => this.useMulticallForType(watcher, type));
+  }
+
+  useMulticallForType(watcher, cdpType) {
```

Comment:
sounds good to me--i just wrote out the code for dealing with a single type first because it was the simplest place to start, but i agree there's no need to expose it as part of the API, so i can refactor to do all types at once